src/device.rs

```diff
@@ -545,7 +549,13 @@ pub unsafe extern "C" fn wgpuBufferMapAsync(
 
 #[no_mangle]
 pub unsafe extern "C" fn wgpuDevicePoll(device: id::DeviceId, force_wait: bool) {
```

Comment:
We should change the signature of this to allow for waiting for a particular submission

Reply:
Oh yeah, we should add a wgpuQueueSubmitForIndex or something that returns a submission index

Reply:
done